src/prettify-type.ts

```diff
@@ -77,3 +83,18 @@ export async function prettifyType (fileName: string, content: string, offset: n
 
   return typeString
 }
+
+const getTsConfigPath = async (fileName: string): Promise<string | undefined> => {
+  const folder = vscode.workspace.getWorkspaceFolder(vscode.Uri.file(fileName))
+
+  if (folder === undefined) return
+
+  const p = path.resolve(folder.uri.fsPath, 'tsconfig.json')
+
+  try {
+    const stat = await fs.stat(p)
+    return stat.isFile() ? p : undefined
+  } catch (e) {
+    return undefined
+  }
+}
```

Comment:
I need to work on file structure more, but for now, would you mind moving this to the function to the helpers file, and import it here?

Other than that, this PR is great, thank you so much for your help on it! This also resolves the bug I was looking into, I really appreciate your help 🙏

Reply:
Apologies - I missed that I can make changes myself. Thanks for your help on this.